src/Compilers/Core/Portable/InternalUtilities/JsonWriter.cs

```diff
@@ -99,51 +97,59 @@ public void Write(string value)
             _output.Write('"');
             _output.Write(EscapeString(value));
             _output.Write('"');
-            _pending = s_commaNewLine;
+            _pending = Pending.CommaNewLineAndIndent;
         }
 
         public void Write(int value)
         {
             WritePending();
-            _output.Write(value);
-            _pending = s_commaNewLine;
+            _output.Write(value.ToString(CultureInfo.InvariantCulture));
```

Comment:
Is this spec'd somewhere? The invariant culture is pretty strange and not common outside of .NET